ubiconfig/_impl/loaders/__init__.py

```diff
@@ -1,4 +1,4 @@
-from .gitlab import GitlabLoader
-from .local import LocalLoader
+from .gitlab import GitlabLoader as _GitlabLoader
+from .local import LocalLoader as _LocalLoader
 
-__all__ = ['GitlabLoader', 'LocalLoader']
+__all__ = ['_GitlabLoader', '_LocalLoader']
```

Comment:
These are already private due to _impl. We can bind them with private names in ubiconfig.ubi, as they are exposed there.

Reply:
I don't like to use import as, it can deceive devs. That's why I rather chose this variant

Reply:
OK. I just mentioned that because you are using import as here.